apps/reference/docs/guides/auth/auth-helpers/auth-ui.mdx

```diff
@@ -36,52 +46,44 @@ const supabase = createClient(
 const App = () => <Auth supabaseClient={supabase} />
 ```
 
-The example above renders the Auth component without any styling.
-
-We recommend using one of the themes provided to style the UI.
+This renders the Auth component without any styling.
+We recommend using one of the predefined themes to style the UI.
 Import the theme you want to use and pass it to the `appearence.theme` prop.
 
 ```diff js title="/src/index.js"
 import {
     Auth,
-+   ThemeSupa
+    // highlight-next-line
```

Comment:
Not sure how I feel about the line highlighting I added in this guide...

Reply:
Also didn't use it for examples that had comments as there's not enough contrast between the comments and the line highlighting:
![Screen Shot 2022-08-22 at 4 07 40 PM](https://user-images.githubusercontent.com/7026076/186034410-c7303d05-8cac-42ac-9b09-9bb0c8825763.png)
![Screen Shot 2022-08-22 at 4 07 49 PM](https://user-images.githubusercontent.com/7026076/186034411-ae0688c5-7b86-4578-b97e-bad02085460f.png)